added data visualisation.

llullabyee opened the pull request:
To do:
- comment on these, add more graphs using means of the entire csv fields
- add instructions to run these in readme

Pull request for what has already been done.

llullabyee committed:
added data visualisation.

To do:
- comment on these, add more graphs using means of the entire csv fields
- add instructions to run these in readme

developing-ar commented on src/constants.py:
```diff
@@ -4,10 +4,13 @@
 import os
 
 RANDOM_SEED = 4403
+DATACOUNT = 261
```
This is unused?

developing-ar reviewed (approved):
Great visualisations of heuristics

llullabyee committed:
added data visualisation.

To do:
- comment on these, add more graphs using means of the entire csv fields
- add instructions to run these in readme
